pkg/arvo/sys/vane/ames.hoon

```diff
@@ -640,11 +675,21 @@
 ::
 +$  ames-state
   $:  peers=(map ship ship-state)
+      =relay-state
       =unix=duct
       =life
       crypto-core=acru:ames
       =bug
   ==
+::  $relay-state: state for a relay
+::
+::    pending:  where to route requests
+::    sponsees: lanes for sponsees
+::
++$  relay-state
+  $:  pending=(map ship [(pair lane (unit lane)) expiry=@da])
+      sponsees=(map ship [lane expiry=@da])
```

Comment:
Wasn't sponsees going to have both rcvr and sndr ships in it?

Reply:
Yeah that was the original design. I noticed that the relays only need to know the destination, so I'm pretty sure it's okay to drop the sender. e.g. if a relay knows the lane of ~zod, it shouldn't matter where the relayed packets are coming from. It just sends them to ~zod's lane.